guava/src/com/google/common/io/Files.java

```diff
@@ -854,7 +845,6 @@ public static String getNameWithoutExtension(String file) {
    *
    * @since 23.5
    */
-  @Beta
```

Comment:
My only hesitation with removing the `@Beta` for `Files.fileTraverser()` is that the `Traverser ` type is itself still beta.

Otherwise, the rest of this commit looks good to me.